livekit-agents/livekit/agents/voice/room_io/_pre_connect_audio.py

```diff
@@ -116,8 +116,13 @@ async def _read_audio_task(self, reader: rtc.ByteStreamReader, participant_id: s
 
             duration: float = 0
 
-            # Check if we need to decode opus
-            is_opus = reader.info.mime_type == "audio/opus"
+            # check if we need to decode opus
+            is_opus = False
+            if reader.info.mime_type:
+                # JS sends "mime_type" as "audio/opus", RN sends it as "audio/webm;codecs=opus"
```

Comment:
```suggestion
                # JS may send "mime_type" as "audio/opus" or "audio/webm;codecs=opus"
```
RN sends it over as "audio/pcm".